package.json

```
@@ -26,6 +26,7 @@
     "@electron/fuses": "^1.8.0",
     "@fortawesome/fontawesome-free": "^6.4.2",
     "@jspsych/plugin-audio-keyboard-response": "^1.1.3",
+    "@jspsych/plugin-call-function": "^2.0.0",
```

Hey! Can you actually set this to be version ^1.1.0? jsPsych just released a new version and all of their plugins have been updated as well, we want the one that's major version #1
```suggestion
    "@jspsych/plugin-call-function": "^2.0.0",
```